exercises/rational-numbers/.meta/src/reference/java/Rational.java

```diff
@@ -0,0 +1,102 @@
+public class Rational {
+    private int numerator;
+    private int denominator;
+
+    Rational(final int numerator, final int denominator) throws IllegalArgumentException {
```

Comment:
`IllegalArgumentException` is an [unchecked exception](https://www.geeksforgeeks.org/checked-vs-unchecked-exceptions-in-java/) therefore you don't have to declare it in the method signature 🙂